api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaExporterTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of the Kafka Exporter deployment.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "image", "groupRegex",
+        "topicRegex", "watchedNamespace",
+        "resources", "logging",
+        "enableSarama", "template"})
+@EqualsAndHashCode
+public class KafkaExporterSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+    public static final int DEFAULT_REPLICAS = 1;
+
+    private int replicas;
```

Comment:
I think replicas do not really make sense here. Are you sure it can even work with multiple replicas? I would mabe remove it similarly to EO.